packages/eas-cli/src/credentials/ios/actions/SetupMultitargetBuildCredentials.ts

```diff
@@ -0,0 +1,94 @@
+import { IosDistributionType, IosEnterpriseProvisioning } from '@expo/eas-json';
+import chalk from 'chalk';
+import nullthrows from 'nullthrows';
+
+import Log from '../../../log';
+import { CredentialsManager } from '../../CredentialsManager';
+import { Context } from '../../context';
+import { IosCapabilitiesOptions } from '../appstore/ensureAppExists';
+import { App, IosAppBuildCredentialsMap, IosCredentials, Target } from '../types';
+import { displayProjectCredentials } from '../utils/printCredentials';
+import { SetupBuildCredentials } from './SetupBuildCredentials';
+
+interface Options {
+  app: App;
+  targets: Target[];
+  distribution: IosDistributionType;
+  enterpriseProvisioning?: IosEnterpriseProvisioning;
+  iosCapabilitiesOptions?: IosCapabilitiesOptions;
+}
+
+export class SetupMultitargetBuildCredentials {
```

Comment:
if this class is used for multi-target and for single target, so maybe it's better to rename it to `SetupBuildCredentials` and old class rename to `SetupTargetBuildCredenitals`